frontend/src/pages/org/archived-item-qa/ui/replay.ts

```diff
@@ -1,29 +1,73 @@
+import { msg } from "@lit/localize";
+import type { SlRequestCloseEvent } from "@shoelace-style/shoelace";
 import { html } from "lit";
 import { guard } from "lit/directives/guard.js";
 import { when } from "lit/directives/when.js";
 
-import type { ReplayData } from "../types";
+import type { QATab, ReplayData } from "../types";
 
 import { renderSpinner } from "./spinner";
 
+import type { Dialog } from "@/components/ui/dialog";
 import { tw } from "@/utils/tailwind";
 
-export function renderReplay(crawlData: ReplayData) {
+export function renderReplay(crawlData: ReplayData, tab: QATab) {
   return html`
-    <div
-      class=${tw`relative h-full overflow-hidden rounded-b-lg border-x border-b bg-slate-100 p-4 shadow-inner`}
-    >
-      ${guard([crawlData], () =>
-        when(
-          crawlData?.replayUrl,
-          (replayUrl) =>
-            html`<iframe
-              src=${replayUrl}
-              class=${tw`h-full w-full overflow-hidden rounded border bg-neutral-0 shadow-lg`}
-            ></iframe>`,
-          renderSpinner,
-        ),
-      )}
+    <div class="replayContainer ${tw`h-full [contain:paint]`}">
+      <div
+        class=${tw`relative h-full overflow-hidden rounded-b-lg border-x border-b bg-slate-100 p-4 shadow-inner md:min-h-[80vh]`}
```

Comment:
Added the 80vh min height, tried `calc()` but it didn't work in the tailwind syntax??  Maybe I was doing something wrong... But either way, good to have a little extra vertical room anyways, seems to work.

Also I think I've ended up liking the extra padding?  Lets keep it for now!

Reply:
Looks like the 80vh min height breaks for laptop-size

Reply:
`min-h-96` seems decent:

<img width="499" alt="Screenshot 2024-05-01 at 4 14 42 PM" src="https://github.com/webrecorder/browsertrix/assets/4672952/c606df4b-ba54-43f4-a339-9d31f7cc5e92">
<img width="883" alt="Screenshot 2024-05-01 at 4 14 54 PM" src="https://github.com/webrecorder/browsertrix/assets/4672952/0e7c43d0-03cd-4c9a-beac-8297e6d0c6a6">